README.md

```diff
@@ -62,11 +62,13 @@ interface {
 ```
 
 To start receive the state and get events, the client needs to send the `start_listening` command.
+See below for info about schemeVersion.
```

Comment:
Should we call it `scheme` or `schema`?

Reply:
I'm not sure...

Reply:
@MartinHjelmare looks like we need to rename to schema:
https://english.stackexchange.com/questions/40702/difference-between-scheme-and-schema

OK ?

Reply:
adjusted it to schema